examples/equilibria/fem_fixed_boundary/fem_magnetostatic_2D_single_coil.py

```diff
@@ -132,16 +119,15 @@
 
 # %%[markdown]
 
-# Convert the mesh in xdmf for reading in fenics. Note that this requires the msh2xdmf
-# module to be available.
+# Convert the mesh in xdmf for reading in fenics.
 
 # %%
 
-msh2xdmf.msh2xdmf("Mesh.msh", dim=2, directory=".")
+msh_to_xdmf("Mesh.msh", dimensions=(0, 2), directory=".")
 
-mesh, boundaries, subdomains, labels = msh2xdmf.import_mesh(
-    prefix="Mesh",
-    dim=2,
+mesh, boundaries, subdomains, labels = import_mesh(
+    "Mesh",
+    dimensions=(0, 2),
```

Comment:
When the files xdmf have been already created, this code would produce the same result for `dimensions=(1,2)`, `dimensions=2`, or `dimensions=(0,1)`. This behavior can be misleading.  A possible solution could be to remove in the `import_mesh` function the possibility to generate the xdmf file:
```
if not os.path.exists(domain_file) or not os.path.exists(boundary_file):
        msh_to_xdmf(f"{file_prefix}.msh", dimensions=dimensions, directory=directory)
```
and to limit `dimensions` to the `mesh.topology().dim()` (i.e. remove `dimensions` from the function signature).